### ISSUE TITLE
image positioning

### ISSUE THREAD

**Issue by stefanseefeld** [nyergler/hieroglyph#145]
Is there a way to control the exact position of embedded images within a given slide, either using (augmented) ReST markup or using CSS ?

**Comment by lpirl** [comment]
Yes. Add a [custom CSS file](http://docs.hieroglyph.io/en/latest/styling.html?highlight=custom%20css#custom-css) to your presentation and then you can use CSS classes for images like this:
```
.. image:: _static/peace.svg
  :class: your-css-class
```
